### DISCUSSION TITLE
Server driven Select component support in clr/angular

### DISCUSSION THREAD

**Discussion by kumar-tadepalli** [vmware/clarity#6211] [Q&A]
Is there an example of Server driven Select component available in clr/angular?

**Comment by mathisscott** [comment]
What do you mean by "server driven select component"? Like a select component that pulls in its options from an API call? 

That seems like something you would use the combobox for... 

Maybe take a look at the "Asynchronous Selection" example on this page: https://v4.clarity.design/documentation/combobox#examples
